crates/data-plane-controller/src/controller.rs

```diff
@@ -671,20 +683,51 @@ impl Controller {
             )
             .await?;
 
+        let branch = if remote == self.infra_remote {
+            &state.deploy_branch
+        } else {
+            "mahdi/data-plane-jsonschema"
```

Comment:
To be updated to `master` after https://github.com/estuary/ops/pull/758 is merged